src/systems/GameSystem.cpp

```diff
@@ -51,12 +51,14 @@ namespace indie
         component2->setType(Component::Type::SPRITE);
         component3->setType(Component::Type::VECTOR);
         component4->setType(Component::Type::EVT_LISTENER);
-        entity2->addComponent(component);
-        entity->addComponent(component2);
-        entity->addComponent(component3);
-        entity2->addComponent(component4);
-        scene->addEntity(entity);
-        scene->addEntity(entity2);
+
+        entity2->addComponent(component)
+            .addComponent(component4);
+
+        entity->addComponent(component2)
```

Comment:
You may use the newly implemented syntax :
```suggestion
        entity->addComponents({component2, component3})
```
to let us see the right way of doing the differents addComponent(s) calls